ydb-platform/ydb-embedded-ui
# feat: show cached data on errors

## Playwright Test Results

**Status**: <span style="color: green; font-weight: bold;">✅ PASSED</span>

| Total | Passed | Failed | Skipped |
|-------|--------|--------|---------|
| 38 | 38 | 0 | 0 |

### 📊 Test Report

For detailed results, please check the [Playwright Report](https://ydb-platform.github.io/ydb-embedded-ui/1095/)

### 🎥 Test Recordings

Video recordings of failed tests (if any) are available in the report.

---

<details>
<summary>ℹ️ How to use this report</summary>

1. Click on the "Playwright Report" link above to view the full test results.
2. In the report, you can see a breakdown of all tests, including any failures.
3. For failed tests, you can view screenshots and video recordings to help debug the issues.
4. Use the filters in the report to focus on specific test statuses or suites.

</details>
